packages/core/src/fetchers/shared/removeCircularReferences.ts

```diff
@@ -0,0 +1,35 @@
+import { ExperienceEntry, Link } from '@/types';
+
+export const removeCircularReferences = (
```

Comment:
Would be nice, if we have a short info about how this function is removing circular references

Reply:
Good call out, I forgot to push that commit ^^